src/app/shared/doc-viewer/doc-viewer.ts

```diff
@@ -91,7 +91,7 @@ export class DocViewer implements OnDestroy {
       let portalHost = new DomPortalHost(
           element, this._componentFactoryResolver, this._appRef, this._injector);
       let examplePortal = new ComponentPortal(componentClass, this._viewContainerRef);
-      let exampleViewer = portalHost.attach(examplePortal);
+      let exampleViewer: any = portalHost.attach(examplePortal);
```

Comment:
Why is this `any`? It should be able to infer the right `ComponentRef`, no?

Reply:
I added `any` here to fix the error on the next line where it was attaching `example` to the instance and not being found. I forgot about this change, didn't mean to commit. I'll fix the type :).